apps/reaper/test/support/test_data/ChargePoint.xml

```diff
@@ -0,0 +1,126 @@
+<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
```

Comment:
Should we commit something this Columbus specific to the general platform repo?

Reply:
I think as long as we're OK to publish this XML then this provides a valuable test case.
  1. It has more complex tags than most of our other tests
  2. It has a block(s) of XML before the block containing the data, so it helps to verify that we can descend and ascend to find the data.